main.cpp

```diff
@@ -545,12 +429,127 @@ MOVE THEM to the space below this block comment and put them in numerical order
     simply CUT and PASTE them in the space provided below:
 */
 
+/*
+Thing 1) Restaurant
+5 properties:
+    1) Number of Menu Items (int)
+    2) Number of Staff Members (int)
+    3) Number of Customers per Week (int)
+    4) Amount of Profit per Week (double)
+    5) Number of Hours Open per Week (int)
+3 things it can do:
+    1) prepare menu item meals
+    2) make money
+    3) pay taxes
 
+Thing 2) Drumset Playing
```

Comment:
You're describing a drummer, not drumset playing.  Drumset Playing is not a noun either.
Correct the name of Thing 2)
- [ ] revise